code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -191,7 +191,10 @@
 	var/mob/camera/aiEye/remote/xenobio/remote_eye = C.remote_control
 	var/obj/machinery/computer/camera_advanced/xenobio/X = target
 
-	if(GLOB.cameranet.checkTurfVis(remote_eye.loc))
+	if(iswallturf(remote_eye.loc))
+		to_chat(owner, "You can't place slime here.")
```

Comment:
Little grammar tweaks. Also needs spans. I suggest `warning` but you can use `notice` as well if you just want the normal blue text.
```suggestion
		to_chat(owner, "<span class='warning'>You can't place a slime here.</span>")
```